### ISSUE TITLE
Is there an issue with Elixir 1.5.x

### ISSUE THREAD

**Issue by arnomi** [PSPDFKit-labs/lager_logger#15]
mix.exs specifically excludes Elixir 1.5 and higher. Is there any particular incompatibility that you know of?

**Comment by MSch** [comment]
No, we just never tested or updated it and went with the conservative choice, in anticipation that the Logger would change a lot more in newer version. If it works for you please send a PR removing the upper limit and I'll do a release if CI is green.
